wdk-ddi-src/content/wdm/nc-wdm-po_fx_directed_power_down_callback.md

```diff
@@ -80,7 +80,7 @@ WDM drivers that register with PoFx for runtime idle power management support ne
 
 Register your implementation of this callback function by setting the appropriate member of the [**PO_FX_DEVICE_V3**](ns-wdm-po_fx_device_v3.md) structure and then calling [**PoFxRegisterDevice**](./nf-wdm-pofxregisterdevice.md).
 
-This callback is invoked after a DRIPS constraint device has been active for at least two consecutive minutes (driver-configurable timeout, default 2 minutes) when the system is in Modern Standby and there is no activator-brokered software activity running.
+This callback is invoked when the system enters DRIPS, not Modern Standby. In other words, this callback is invoked after a DRIPS constraint device has been active for at least two consecutive minutes (driver-configurable timeout, default 2 minutes) when the system is in Modern Standby and there is no activator-brokered software activity running.

```

Comment:
It would be a bit more accurate to say "This callback is invoked during a Modern Standby session when the system has entered DRIPS, and a device in the DRIPS constraint path has been active for at least two consecutive minutes (driver-configurable timeout, default 2 minutes) without any activator-brokered software activity running."

This clarifies that it does not fire immediately on DRIPS entry. It also clarifies that the device may be a child of a constraint device, not necessarily a constraint device itself.